osu.Game.Rulesets.Catch/Beatmaps/CatchBeatmapProcessor.cs

```diff
@@ -15,19 +15,26 @@ namespace osu.Game.Rulesets.Catch.Beatmaps
 {
     public class CatchBeatmapProcessor : BeatmapProcessor
     {
+        public const int RNG_SEED = 1337;
+
         public CatchBeatmapProcessor(IBeatmap beatmap)
             : base(beatmap)
         {
         }
 
-        public override void PostProcess()
```

Comment:
I think catch hyperdash initialization should be a post process after nested objects made because a big droplet can be hyper (this is unimplemented though).

Reply:
Yeah sounds fair, I considered this.